wiki/Client/Release_stream/Lazer/Feature_comparison/en.md

```diff
@@ -0,0 +1,438 @@
+
+This kind of makes sense when you look at it, but adds complexity to computing gameplay (and potentially adds replay errors, especially with higher OD values and slider velocity).
+
+The rationale behind this change is simple: without it, having `Snaking out sliders` enabled becomes very awkward to visualise when a player hits late. Do you leave the original circle in place and delay the outward snaking? How does it catch up to the actual location of the slider ball if so? And on the contrary, if you don't delay the snaking, do you leave the head circle detached from the slider (please no)?
+
+|  |  |
+| :-- | :-: |
+| Breaks backwards compatibility | Yes |
+| Classic mod revert support | Yes |
+| Intentionally changed | Yes |
+| Needs further consideration | Yes |
+
+#### Slider ends do not contribute to combo nor cause misses
+
+Slider ends used to have the quirk of not breaking combo (and not causing a MISS judgement) if missed. This meant that an S play, which among other things requires zero MISS judgements, would look like a full combo even if some slider ends were missed. Various debates over whether such a play should be called an FC ensued.
+
+From now on slider ends will only affect score and accuracy, but have no effect on combo. This means that an S grade always signifies a full combo.
+
+|  |  |
+| :-- | :-: |
+| Breaks backwards compatibility | Yes |
+| Classic mod revert support | Yes |
+| Intentionally changed | Yes |
+| Needs further consideration | Yes |
+
+#### Missing a slider head causes a miss
+
+Missing a slider head (either by not hitting it or hitting it during its miss window) would previously break combo but not cause a MISS judgement, and a judgement could still be received for the missed slider by completing the rest of it. This allowed players to receive the S grade while having a low max combo.
+
+In lazer, not hitting the slider head will give a MISS judgement for the whole slider. After missing a slider head, combo, score, and accuracy can still be gained from slider ticks and repeats, while only score and accuracy can still be gained from the slider end.
+
+|  |  |
+| :-- | :-: |
+| Breaks backwards compatibility | Yes |
+| Classic mod revert support | No |
+| Intentionally changed | Yes |
+| Needs further consideration | No |
+
+#### Slider ends do not cause hitsounds when not hit
+
+In stable, slider ends would play their hitsounds even if they were missed, as long as any part of the slider was hit. This has been changed such that hitsounds match inputs 1:1.
+
+|  |  |
+| :-- | :-: |
+| Breaks backwards compatibility | No |
+| Classic mod revert support | Yes |
+| Intentionally changed | Yes |
+| Needs further consideration | No |
+
+#### The spinner spin speed cap of 477 RPM has been removed
+
+Instead of a speed cap, spinners now have a score cap. The reason for this is inconsistency in how score is awarded, because the algorithm is dependent on frame rate.
```

Comment:
That's not really the reason. It was just a simple implementation we haven't done another pass on yet. Calculations on lazer are done in a way that frame rate should not be an issue, at very least. But to make spinners feel good it may factor back in, we'll see.

Reply:
hmm this was something i vaguely recalled hearing from some dev discussion, but can't find it anywhere. either way, it is true that frame rate affects spinning on stable, but yes i can remove it if it wasn't the actual reasoning.

Reply:
reworded